$OpenBSD$

WIP. It's inteded to fix the build on a mac ppc. It doesn't fix it
yet - I need to implement getting time base_frequency.

Index: absl/base/internal/unscaledcycleclock.cc
--- absl/base/internal/unscaledcycleclock.cc.orig
+++ absl/base/internal/unscaledcycleclock.cc
@@ -26,6 +26,10 @@
 #elif defined(__FreeBSD__)
 #include <sys/sysctl.h>
 #include <sys/types.h>
+#elif defined(__OpenBSD__)
+#include "absl/base/call_once.h"
+#include <sys/types.h>
+#include <sys/sysctl.h>
 #endif
 #endif
 
